### DISCUSSION TITLE
Spotifyd Output device not applied or saved; always reverts to hw:0,0

### DISCUSSION THREAD

**Discussion by TinkeringB** [rern/rAudio#1452] [Bug Reports]
![image](https://github.com/rern/rAudio/assets/9662310/1ce23468-1746-4cf7-92c0-54b38e2646f1)

then 
![image](https://github.com/rern/rAudio/assets/9662310/7ba954c0-1289-44e4-9a1b-8f437d12c3ac)

I need the selection to change to "default:CARD=A20", but rAudio will not let me. So now I have to manually edit /etc/spotifyd.conf after each reboot when I want to listen to Spotify.

rAudio 20231013
6.1.53-1 aarch64
Raspberry Pi 3B Rev 1.2
BCM2837 • 907 MiB
4 x Cortex-A53 @ 1.20 GHz

**Comment by rern** [comment]
Please try:
`sed -i 's#|].*//g#\(.*\)],.*/\\1/#' $dirsettings/features.sh`

**Reply by TinkeringB** [reply]
Thank you @rern ! - that seems to do the trick :-)
